### DISCUSSION TITLE
Mounting folders for training

### DISCUSSION THREAD

**Discussion by simonmcnair** [AbdBarho/stable-diffusion-webui-docker#492] [General]
I have noticed quite a few of the folders get blown away on a rebuild that don't need to be.

```/models
drwxr-xr-x 2 root root 4096 May 26 12:18 Autoprune
drwxr-xr-x 2 root root 4096 May 26 12:18 Components
drwxr-xr-x 2 root root 4096 May 26 12:18 Deforum
drwxr-xr-x 2 root root 4096 May 15 22:05 VAE-approx
drwxr-xr-x 2 root root 4096 May 15 22:05 deepbooru
drwxr-xr-x 2 root root 4096 May 26 12:32 deepdanbooru
drwxr-xr-x 2 root root 4096 May 26 12:32 dreambooth
drwxr-xr-x 2 root root 4096 May 15 22:08 karlo
drwxr-xr-x 2 root root 4096 May 30 09:39 leres
drwxr-xr-x 2 root root 4096 May 30 09:54 pix2pix```

I also think it might be handy to create and mount an empty training directory ?

I currently bind mount /training to a training folder in my docker directory

I also can't see textual_inversions unless I go to console.

**Comment by AbdBarho** [comment]
What do these folders contain? are they empty?

For the training, you could use the `data` folder which is already mounted under `/data`?

**Reply by simonmcnair** [reply]
When you initially start training, doing danburu or captioning things A1111 automatically downloads different models of things.

**Reply by AbdBarho** [reply]
I see, I have such an old pc that I did not even bother trying the training process.

Do you have an idea how much data is downloaded to these folders?

**Reply by simonmcnair** [reply]
Please can you merge my pull ? https://github.com/AbdBarho/stable-diffusion-webui-docker/pull/505

It will add the folders neccessary.  If people don't use them, it won't cause any issues, they'll just have some empty folders.
Cheers

**Comment by AbdBarho** [comment]
This should be fixed with https://github.com/AbdBarho/stable-diffusion-webui-docker/releases/tag/8.0.0
